hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientMembershipListener.java

```diff
@@ -94,7 +94,7 @@ public void handle(Set<Member> initialMembers) {
 
         final List<MembershipEvent> events = detectMembershipEvents(prevMembers);
         if (events.size() != 0) {
-            applyMemberListChanges();
+            LOGGER.info(membersString());
```

Comment:
could we add more context to the log? i.e got initial members
Same as the other logging